assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml

```diff
@@ -24,4 +24,4 @@ tags:
   license: apache-2.0
   task: chat-completion
   author: mistralai
-version: 1
+version: 2
```

Comment:
Did you verify this model works with the current container?
This is new model added in Jan, can you double check if you haven't already

Reply:
will do it separately, wanted to focus on image update for this pr

Reply:
don't forget to update this one to 28, I don't see it updated yet in the blob.